tests/conftest.py

```diff
@@ -44,9 +46,10 @@ def entry_xml(self):
               </content>
             </entry>
         """
-        self.attributes["event_date_time"] = xsDateTime_format(
-            self.attributes["event_date_time"]
-        )
+        if isinstance(self.attributes['event_date_time'], datetime.datetime):
+            self.attributes["event_date_time"] = xsDateTime_format(
+                self.attributes["event_date_time"]
+            )
```

Comment:
This `if` and this file has inconsistent single quote/double quote usage around `attributes` keys